network-config-analyzer/NetworkConfig.py

```diff
@@ -51,6 +51,8 @@ def __init__(self, name, peer_container, entry_list=None, config_type=None):
         self.referenced_ip_blocks = None
         self.type = config_type or NetworkConfig.ConfigType.Unknown
         self.allowed_labels = set()
+        if from_buffer:
+            entry_list = 'buffer: ' + entry_list
         peer_container.clear_pods_extra_labels()
         for entry in entry_list or []:
             self.add_policies_from_entry(entry)
```

Comment:
```suggestion
        peer_container.clear_pods_extra_labels()
        if from_buffer:
            entry_list = 'buffer: ' + entry_list
        else:
            for entry in entry_list or []:
                self.add_policies_from_entry(entry)
```
Sorry, I wasn't clear enough. Here is my suggestion.